test/nodes/core/io/22-websocket_spec.js

```diff
@@ -339,6 +339,18 @@ describe('websocket Node', function() {
             });
         });
 
+        it('should handle protocol property', function(done) {
```

Comment:
This just tests you can set the parameter - doesn't actually test it does anything,